### PR TITLE
[AIR-2892] Modify the SDK so that users can use public apps.

### PR DESCRIPTION
**Pull request by stabarez**
Modify the SDK so that users can use public apps.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,6 +12,10 @@
     "README.md",
     "package.json"
   ],
+  "repository": {
```
Comment (stabarez) [inline package.json:15]: This is for adding the repository link in npm.

**Review by bernabe9 — APPROVED** [review]

**Review by vickymadrid03 — APPROVED** [review]

File: src/AirOps.ts [hunk 2]
```diff
@@ -7,12 +7,14 @@ class AirOps implements AirOpsInterface {
   private readonly hashedUserId?: string;
   private readonly host?: string;
 
-  constructor(params: IdentifyParams) {
-    const { userId, workspaceId, hashedUserId, host } = params;
-    this.workspaceId = workspaceId;
-    this.userId = userId;
-    this.hashedUserId = hashedUserId;
-    this.host = host || 'https://app.airops.com';
+  constructor(params?: IdentifyParams) {
```
Comment (Tintef) [inline src/AirOps.ts:10]: I think we can have two constructors instead

File: src/AirOps.ts [hunk 3]
```diff
@@ -38,10 +40,6 @@ class AirOps implements AirOpsInterface {
    * @returns - Apps instance
    */
   public get apps(): AirOpsApps {
-    if (!this.workspaceId || !this.userId || !this.hashedUserId) {
-      throw new Error('You must identify before using the SDK');
-    }
```
Comment (Tintef) [inline src/AirOps.ts:43]: Is it possible to list apps without auth?

Comment (stabarez) [reply src/AirOps.ts:43]: Nop, we don't have list apps yet, should we added?

**Review by Tintef — APPROVED** [review]

**Pull request merged**